### DISCUSSION TITLE
Telegram Floating Button

### DISCUSSION THREAD

**Discussion by Dongarsi** [elementor/elementor#28090] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

The floating buttons feature you added was really necessary and works perfectly. The only thing is that you have added all the contact ways except Telegram! 

### Describe the solution you'd like.

This function would be complete and perfect if it also included telegram or left a possibility of customization with an empty field, in which to choose custom icon and links. Otherwise it would be enough to add Telegram

### Describe alternatives you've considered.

This function would be complete and perfect if it also included telegram or left a possibility of customization with an empty field, in which to choose custom icon and links. Otherwise it would be enough to add Telegram

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by kavs777** [comment]
+1
